package.json

```diff
@@ -10,12 +10,12 @@
   "scripts": {
     "build": "cross-env NODE_ENV=production npm run build-lib && npm run build-dist",
     "watch": "npm run watch-components",
-    "build-lib": "lerna run build --parallel",
-    "build-dist": "lerna run build-dist --parallel",
-    "watch-components": "lerna run watch --parallel",
+    "build-lib": "lerna run build",
+    "build-dist": "lerna run build-dist",
+    "watch-components": "lerna run watch",
     "setup-npm-links": "node scripts/setup-npm-link.js",
     "postinstall": "lerna bootstrap --hoist && cross-env NODE_ENV=production npm run build-lib && lerna run --parallel genPackagelock && playwright install --with-deps chromium",
-    "release": "npm whoami && lerna clean --yes && git clean -dfxq -e .idea -e .vscode && npm ci && cross-env NODE_ENV=production npm run build-dist && lerna version --exact && lerna publish from-package && git clean -dfxq -e .idea -e .vscode && npm install && git add -A && git commit -m \"release: update package locks\" && git push",
+    "release": "npm whoami && lerna clean --yes && git clean -dfxq -e .idea -e .vscode && npm ci && cross-env NODE_ENV=production npm run build-dist && lerna version --include-merged-tags --exact && lerna publish from-package && git clean -dfxq -e .idea -e .vscode && npm install && git add -A && git commit -m \"release: update package locks\" && git push",
```

Comment:
in 1 sentence: is the new cmd-line param just a rename or does it do something new?